traffic_portal/app/src/common/modules/table/roleUsers/table.roleUsers.tpl.html

```diff
@@ -64,6 +65,7 @@
                 <td data-search="^{{::u.tenant}}$">{{::u.tenant}}</td>
                 <td data-search="^{{::u.rolename}}$">{{::u.rolename}}</td>
                 <td data-search="^{{::u.registrationSent}}$">{{::u.registrationSent}}</td>
+                <td data-search="^{{::getRelativeTime(u.lastAuthenticated)}}$">{{::getRelativeTime(u.lastAuthenticated)}}</td>
```

Comment:
`getRelativeTime` doesn't handle null values, so if a user has never logged in the table will display "Invalid date" as their last authentication time. It should say "never" or similar.

This will happen whenever a new user is created or registered until they then log in.

Reply:
Done